stable/kubexray/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # JFrog KubeXray Chart Changelog
 All changes to this chart will be documented in this file.
 
-## [0.3.3] - Mar 13, 2019
-* Move securityContext to container level
+## [0.3.4] - Mar 15, 2019
+* Revert securityContext change that was causing issues
```

Comment:
@danielezer Same here add changelog for 0.3.3

Reply:
yup, done